src/test/java/com/google/cloud/teleport/splunk/CustomX509TrustManagerTest.java

```diff
@@ -47,13 +47,11 @@ public void setUp()
         new FileInputStream(classLoader.getResource("PubsubToSplunkTestData/RootCA.crt").getFile());
     FileInputStream recognizedInputStream =
         new FileInputStream(
-            classLoader
-                .getResource("PubsubToSplunkTestData/RootCASignedCertificate.crt")
-                .getFile());
```

Comment:
It seems that the PR includes changes merged from the main branch.
And [ggprod:elasticsearch-io-improvements](https://github.com/ggprod/DataflowTemplates/tree/elasticsearch-io-improvements) branch is also behind `main`.
Can you please **rebase** the branch onto master? (**merge** won't work as our CLA check won't accept commits made by `cloud-teleport` in your PR).